IBPSA/Fluid/FixedResistances/Examples/PlugFlowPipeNoMix.mo

```diff
@@ -0,0 +1,26 @@
+within IBPSA.Fluid.FixedResistances.Examples;
+model PlugFlowPipeNoMix
```

Comment:
Done. Although I find merging two variation of the same examples a bit awkward since it requires to duplicate part or all of the other classes that are connected to it. I know that they are work around it (maybe creating a base class, or by vectorizing the other components), but it does create example models that are not as simple to read.